src/dispensing.component.tsx

```diff
@@ -7,7 +7,7 @@ export default function Dispensing() {
   return (
     <div className={`omrs-main-content`}>
       <PharmacyHeader />
-      <DispensingTiles />
+      {/*      <DispensingTiles />*/}
```

Comment:
Just commenting this out this I don't think these tabs will be implemented for v1 and I wanted to avoid having people reviewing the UI getting confused by them until they are implemented.